docs/manual/java/guide/services/ServiceErrorHandling.md

```diff
@@ -9,6 +9,9 @@ There are a number of principles behind the design of Lagom's built in error han
 * If possible, Lagom will try to reconstruct errors on the client side when thrown on the service side.  So, if the server side throws an exception saying it couldn't serialize something, the client code should receive that same exception.
 * If possible, exceptions should be mapped to idiomatic protocol response codes, such as HTTP 4xx and 5xx status codes and WebSocket error close codes.
 
+
+If you are using Lagom to consume a service (either implemented in Lagom or a third-party stack) the client Lagom provides will map responses with status code values in the ranges 4xx and 5xx to exceptions. That has an impact on the [[Circuit Breakers|ServiceClients#Circuit-Breakers]] the client is using to connect to that endpoint. By default Lagom Circuit Breakers will account any exception as an error but that behavior is [[configurable|ServiceClients#Circuit-Breaker-Configuration]]. So 4xx and 5xx will be mapped to exceptions but you can whitelist what exceptions should not trip the circuit breaker.
```

Comment:
RE: "That has an impact on the Circuit Breakers the client is using to connect to that endpoint." Does a client connect using a Circuit Breaker, or should this sentence be reworded slightly, such as "That has an impact on the Circuit Breaker for that endpoint." 

Then maybe: By default, Circuit Breakers count all exceptions as errors, but that behavior is configurable. You can whitelist 4xx and 5xx status codes so they do not trip the Circuit Breaker.

Reply:
> "You can whitelist 4xx and 5xx status codes so they do not trip the Circuit Breaker."

Well, the thing is that you don't whitelist the status codes, you whitelist the exceptions. I rewrote that last part into: "So 4xx and 5xx will be mapped to exceptions but you can whitelist what exceptions should not trip the circuit breaker."

Reply:
>  Does a client connect using a Circuit Breaker, or should this sentence be reworded slightly,

A client may connect via a Circuit breaker. I could be a bad citizen and use a custom client that has no circuit breaking. This documentation refers to the client and circuit breakers provided by lagom in which case the client has a circuit breaker.